cpp/src/arrow/adapters/orc/adapter.cc

```diff
@@ -173,7 +173,7 @@ class OrcStripeReader : public RecordBatchReader {
   int64_t batch_size_;
 };
 
-liborc::RowReaderOptions default_row_reader_options() {
+liborc::RowReaderOptions DefaultRowReaderOptions() {
```

Comment:
This function was added by me in the past and now I have changed its name to follow the same style.